plugins/speculation-rules/tests/test-speculation-rules.php

```diff
@@ -7,69 +7,15 @@
 
 class Test_Speculation_Rules extends WP_UnitTestCase {
 
-	/** @var array<string, mixed> */
-	private $original_wp_theme_features = array();
-
-	public function set_up(): void {
-		parent::set_up();
-		$this->original_wp_theme_features = $GLOBALS['_wp_theme_features'];
-	}
-
-	public function tear_down(): void {
-		$GLOBALS['_wp_theme_features'] = $this->original_wp_theme_features;
-		parent::tear_down();
-	}
-
 	public function test_hooks(): void {
-		$this->assertSame( 10, has_action( 'wp_footer', 'plsr_print_speculation_rules' ) );
-		$this->assertSame( 10, has_action( 'wp_head', 'plsr_render_generator_meta_tag' ) );
-	}
-
-	/**
-	 * @covers ::plsr_print_speculation_rules
-	 */
-	public function test_plsr_print_speculation_rules_without_html5_support(): void {
```

Comment:
Note for reviewers: These tests for the `plsr_print_speculation_rules()` function were moved to `test-speculation-rules-plugin-api.php` without any changes.